changelog/unreleased/kong/bump-lua-messagepack-0.5.3.yml

```
@@ -0,0 +1,3 @@
+message: "Bumped OpenResty from 0.5.2 to 0.5.3"
```

Hmmm shouldn't it be:

```suggestion
message: "Bumped lua-messagepack from 0.5.2 to 0.5.3"
```
?